Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,511 @@
+
+        QWidget* secondScreenScrollWidget = new QWidget();
+        secondScreenScrollArea->setWidget(secondScreenScrollWidget);
+
+        QVBoxLayout* secondScreen = new QVBoxLayout(this);
+        secondScreen->setAlignment(Qt::AlignTop);
+        secondScreen->addWidget(secondScreenFrame);
+        secondScreenScrollWidget->setLayout(secondScreen);
+
+        QLabel* secondRightPaneHeader = new QLabel();
+        secondRightPaneHeader->setObjectName("rightPaneHeader");
+        secondRightPaneHeader->setText(tr("Enter Gem Details"));
+        secondScreen->addWidget(secondRightPaneHeader);
+
+        m_gemDisplayName = new FormLineEditWidget(tr("Gem Display name*"), "");
+        QLineEdit* gemDisplayLineEdit = m_gemDisplayName->lineEdit();
+        gemDisplayLineEdit->setPlaceholderText(tr("The name displayed on the Gem Catalog"));
+        secondScreen->addWidget(m_gemDisplayName);
+
+        m_gemSystemName = new FormLineEditWidget(tr("Gem System name*"), "");
+        QLineEdit* gemSystemLineEdit = m_gemSystemName->lineEdit();
+        gemSystemLineEdit->setPlaceholderText(tr("The name used in the O3DE Gem System"));
+        secondScreen->addWidget(m_gemSystemName);
+
+        m_gemSummary = new FormLineEditWidget(tr("Gem Summary"), "");
+        QLineEdit* gemSummaryLineEdit = m_gemSummary->lineEdit();
+        gemSummaryLineEdit->setPlaceholderText(tr("A short description of your Gem"));
+        secondScreen->addWidget(m_gemSummary);
+
+        m_requirements = new FormLineEditWidget(tr("Requirements"), "");
+        QLineEdit* requirementsLineEdit = m_requirements->lineEdit();
+        requirementsLineEdit->setPlaceholderText(tr("Notice of any requirements your Gem. i.e. This requires X other gem"));
+        secondScreen->addWidget(m_requirements);
+
+        m_license = new FormLineEditWidget(tr("License*"), "");
+        QLineEdit* licenseLineEdit = m_license->lineEdit();
+        licenseLineEdit->setPlaceholderText(tr("License uses goes here: i.e. Apache-2.0 or MIT"));
+        secondScreen->addWidget(m_license);
+
+        m_licenseURL = new FormLineEditWidget(tr("License URL"), "");
+        QLineEdit* licenseURLLineEdit = m_licenseURL->lineEdit();
+        licenseURLLineEdit->setPlaceholderText(tr("Link to the license web site i.e. https://opensource.org/licenses/Apache-2.0"));
+        secondScreen->addWidget(m_licenseURL);
+
+        m_origin = new FormLineEditWidget(tr("Origin"), "");
+        QLineEdit* originLineEdit = m_origin->lineEdit();
+        originLineEdit->setPlaceholderText(tr("The name of the originator goes. i.e. XYZ Inc."));
+        secondScreen->addWidget(m_origin);
+
+        m_originURL = new FormLineEditWidget(tr("Origin URL"), "");
+        QLineEdit* originURLLineEdit = m_originURL->lineEdit();
+        originURLLineEdit->setPlaceholderText(tr("The primary repo for your Gem. i.e. http://www.mydomain.com"));
+        secondScreen->addWidget(m_originURL);
```

Comment:
The `Origin` is actually the same as the creator.  These two fields should be moved to the third tab.